packages/scandipwa/src/component/CheckoutAddressBook/CheckoutAddressBook.component.js

```diff
@@ -31,20 +31,27 @@ export class CheckoutAddressBook extends PureComponent {
         onShippingEstimationFieldsChange: PropTypes.func.isRequired,
         selectedAddressId: PropTypes.number.isRequired,
         isBilling: PropTypes.bool.isRequired,
-        isSubmitted: PropTypes.bool
+        isSubmitted: PropTypes.bool,
+        is_virtual: PropTypes.bool
     };
 
     static defaultProps = {
-        isSubmitted: false
+        isSubmitted: false,
+        is_virtual: false
     };
 
     state = {
         isCustomAddressExpanded: false
     };
 
     static getDerivedStateFromProps(props) {
-        const { selectedAddressId } = props;
+        const { is_virtual, selectedAddressId } = props;
+
         if (selectedAddressId === 0) {
+            if (is_virtual) {
```

Comment:
Can be simplified: 
`return is_virtual ? { isCustomAddressExpanded: true } : null`

Reply:
Guys... show the address form if is_virtual?
Shouldn't it be the opposite?